src/client/runner/automation/playback/press/index.js

```diff
@@ -141,6 +153,18 @@ export default class PressAutomation {
     }
 
     run () {
+        var activeElement = domUtils.getActiveElement();
+        var isIframe      = domUtils.isIframeElement(activeElement);
```

Comment:
`isElementInIframe` means that `element` is placed within an `iframe`, but actually it's a check that `element` *is Iframe instance*. Maybe something like `activeElementIsIframe`, but it doesn't starts with `it`.